### ISSUE TITLE
Failed at the hashring@3.1.1 install script 'node-gyp rebuild'

### ISSUE THREAD

**Issue by AleCaste** [3rd-Eden/memcached#246]
Hello all,
I have latest node version on Windows 7 (node v0.12.3) and latest npm version (npm 2.10.0).
I also have Python installed on the machine and I already registered it:
```npm config set python D:\python```
... but when I try to install memcached client on my node project I get the following error stacktrace:
```
npm install memcached --save
|
> hashring@3.1.1 install D:\xampp\htdocs\exp0\node_modules\memcached\node_modules\hashring
> node-gyp rebuild


D:\xampp\htdocs\exp0\node_modules\memcached\node_modules\hashring>if not defined npm_config_node_gyp (node "C:\Users\Alejandro\AppData\Roaming\npm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (rebuild)
gyp ERR! configure error
gyp ERR! stack Error: spawn D:\python ENOENT
gyp ERR! stack     at exports._errnoException (util.js:746:11)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1053:32)
gyp ERR! stack     at child_process.js:1144:20
gyp ERR! stack     at process._tickCallback (node.js:355:11)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "C:\\Users\\Alejandro\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd D:\xampp\htdocs\exp0\node_modules\memcached\node_modules\hashring
gyp ERR! node -v v0.12.3
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "node" "C:\\Users\\Alejandro\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "memcached" "--save"
npm ERR! node v0.12.3
npm ERR! npm  v2.10.0
npm ERR! code ELIFECYCLE

npm ERR! hashring@3.1.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hashring@3.1.1 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the hashring package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls hashring
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\xampp\htdocs\exp0\npm-debug.log
```
Any idea about how to overcome this?
Thanks a lot

**Comment by rustybailey** [comment]
I ran into this error today and found this issue through Google. I ran again with `npm install --msvs_version=2012` and that worked for me.

**Comment by Gorokhov** [comment]
doesn't work for me, any other options, how can i help detecting the issue?

**Comment by rexxars** [comment]
It seems the latest version of hashring uses a plain javascript implementation, and that `memcached` was updated to use it. However, no new version seems to have been released yet.

**Comment by 3rd-Eden** [comment]
Published fix as 2.2.0
